packages/office-ui-fabric-react/src/components/Rating/Rating.scss

```diff
@@ -0,0 +1,85 @@
+@import '../../common/common';
+
+$defaultColor: $ms-color-neutralTertiary;
+$selectedColor: $ms-color-black;
+$hoverColor: $ms-color-themeDarkAlt;
+$disabledColor: $ms-color-neutralTertiaryAlt;
+
+// Hide input field
+.ms-Rating-input {
```

Comment:
Radio buttons are the right call here. They represent exclusive selection in a list of items which they do. Simple buttons do not support that. It's all about the functionality and not appearance.